docs/src/content/docs/resources/plugins.mdx

```diff
@@ -110,6 +110,11 @@ A theme is a Starlight plugin that changes the visual appearance of a site with
 		title="starlight-theme-obsidian"
 		description="Starlight theme inspired by the style of Obsidian Publish sites."
 	/>
+	<LinkCard
+		href="https://github.com/TheOtterlord/catppuccin-starlight"
+		title="catppuccin-starlight"
+		description="📝 Soothing pastel theme for Starlight"
```

Comment:
I know this is the repo description but given nothing else uses an emoji here, I'd suggest we leave it out. For fear of starting an emoji arms race between plugins 😄 

```suggestion
		description="Soothing pastel theme for Starlight"
```

Reply:
Haha, fair enough. It's the standard tagline template for catppuccin, so I just copy-pasted.